### PR TITLE
feat(core): Rename account-link doctype to caip10-link

### PR DESCRIPTION
**Pull request by stbrody**

### CODE REVIEW

File: packages/ceramic-core/src/__tests__/__validation__/__schemas__/crypto-account-links.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object",
-  "title": "CryptoAccountLinks",
+  "title": "CryptoAccountCaip10Links",
```
Comment (stbrody) [inline packages/ceramic-core/src/__tests__/__validation__/__schemas__/crypto-account-links.json:4]: Wasn't totally sure if this change made sense, lmk if I should put it back the way it was

Comment (oed) [reply packages/ceramic-core/src/__tests__/__validation__/__schemas__/crypto-account-links.json:4]: Not sure if these schemas are still needed here. Are they used for testing maybe @simonovic86 ?
Anyhow, I'd just name this `CryptoAccounts`.

**Review by oed — APPROVED** [review]
Looks good

File: packages/ceramic-core/src/__tests__/__validation__/__schemas__/crypto-account-links.json (hunk 1 above)
Comment (simonovic86) [reply packages/ceramic-core/src/__tests__/__validation__/__schemas__/crypto-account-links.json:4]: @oed yep, they are used for testing only 👍

**Review by simonovic86 — APPROVED** [review]
LGTM

**Pull request merged**